### ISSUE TITLE
[Semantics] SingleChildScrollView requests focus in accessibility mode in iOS

### ISSUE THREAD

**Issue by NilouMinooei** [flutter/flutter#164623]
### Steps to reproduce

1. in the app have a long scrollable column and add a SingleChildScrollView over the column,
something like this (make sure the number of elements are sufficient, it needs to be scrollable):
```dart
SingleChildScrollView(
          child: Column(
            mainAxisAlignment: MainAxisAlignment.center,
            children: <Widget>[
              const Text('You have pushed the button this many times:'),
              Text('$_counter', style: Theme.of(context).textTheme.headlineMedium),
              ...List.generate(100, (index) {
                return Padding(padding: EdgeInsets.all(8),child: Text('and next($index) text here'));
              }),
            ],
          ),
        ),
```
2. start the app on iOS real device with voice over on
3. use normal slide gesture to navigate on the column view
 
Note: I tried it with 2 other versions of flutter 3.27.4 and 3.24.4 and in then this issue did not happen. So 2 points, make sure you have flutter 3.29 and the column is scrollable.

### Expected results

After sliding from title, the selected element should be the first element:

<img width="396" alt="Image" src="https://github.com/user-attachments/assets/0b112a7d-e31e-4c64-ac1f-c6f40bf95937" />

### Actual results

Right now, after sliding from title the whole column gets selected:

<img width="393" alt="Image" src="https://github.com/user-attachments/assets/5b9c25e1-e40b-4b4d-9619-8f1124aecf28" />

### Code sample

<details open><summary>Code sample</summary>

```dart
import 'package:flutter/material.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({super.key});

  // This widget is the root of your application.
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple)),
      home: const MyHomePage(title: 'Flutter Demo Home Page'),
    );
  }
}

class MyHomePage extends StatefulWidget {
  const MyHomePage({super.key, required this.title});

  final String title;

  @override
  State<MyHomePage> createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  int _counter = 0;

  void _incrementCounter() {
    setState(() {
      _counter++;
    });
  }

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(backgroundColor: Theme.of(context).colorScheme.inversePrimary, title: Text(widget.title)),
      body: Center(
        child: SingleChildScrollView(
          child: Column(
            mainAxisAlignment: MainAxisAlignment.center,
            children: <Widget>[
              const Text('You have pushed the button this many times:'),
              Text('$_counter', style: Theme.of(context).textTheme.headlineMedium),
              ...List.generate(100, (index) {
                return Padding(padding: EdgeInsets.all(8),child: Text('and next($index) text here'));
              }),
            ],
          ),
        ),
      ),
      floatingActionButton: FloatingActionButton(
        onPressed: _incrementCounter,
        tooltip: 'Increment',
        child: const Icon(Icons.add),
      ),
    );
  }
}

```

</details>

### Screenshots or Video

<details open>
<summary>Screenshots / Video demonstration</summary>
what it should be (like older flutter versions):
https://github.com/user-attachments/assets/396659c4-483f-4c64-afd2-38ea30b9322b

what it is(flutter 3.29):
https://github.com/user-attachments/assets/2feaf622-8a36-4d92-9627-846822ca5d50

</details>

### Flutter Doctor output

<details open><summary>Doctor output</summary>

```console
[✓] Flutter (Channel stable, 3.29.0, on macOS 15.3.1 24D70 darwin-arm64, locale en-DE) [2.0s]
    • Flutter version 3.29.0 on channel stable at /Users/nmi/fvm/versions/3.29.0
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 35c388afb5 (3 weeks ago), 2025-02-10 12:48:41 -0800
    • Engine revision f73bfc4522
    • Dart version 3.7.0
    • DevTools version 2.42.2

[✓] Android toolchain - develop for Android devices (Android SDK version 35.0.0-rc4) [4.3s]
    • Android SDK at /Users/xx/Library/Android/sdk
    • Platform android-35, build-tools 35.0.0-rc4
    • ANDROID_HOME = /Users/xx/Library/Android/sdk
    • ANDROID_SDK_ROOT = /Users/xx/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
      This is the JDK bundled with the latest Android Studio installation on this machine.
      To manually set the JDK path, use: `flutter config --jdk-dir="path/to/jdk"`.
    • Java version OpenJDK Runtime Environment (build 17.0.10+0-17.0.10b1087.21-11609105)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 16.2) [4.8s]
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 16C5032a
    • CocoaPods version 1.16.2

[✗] Chrome - develop for the web (Cannot find Chrome executable at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome) [13ms]
    ! Cannot find Chrome. Try setting CHROME_EXECUTABLE to a Chrome executable.

[✓] Android Studio (version 2024.1) [13ms]
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.10+0-17.0.10b1087.21-11609105)

[✓] VS Code (version 1.97.2) [10ms]
    • VS Code at /Users/nmi/Desktop/Visual Studio Code.app/Contents
    • Flutter extension version 3.106.0

[✓] Connected device (3 available) [8.6s]
    • iPhone von Mobile (mobile)      • 00008020-001905A00C82002E • ios           • iOS 18.3.1 22D72
    • macOS (desktop)                 • macos                     • darwin-arm64  • macOS 15.3.1 24D70 darwin-arm64
    • Mac Designed for iPad (desktop) • mac-designed-for-ipad     • darwin        • macOS 15.3.1 24D70 darwin-arm64
    ! Error: Browsing on the local area network for iPhone. Ensure the device is unlocked and attached with a cable or associated with the same
      local area network as this Mac.
      The device must be opted into Developer Mode to connect wirelessly. (code -27)
    ! Error: Browsing on the local area network for iPhone. Ensure the device is unlocked and attached with a cable or associated with the same
      local area network as this Mac.
      The device must be opted into Developer Mode to connect wirelessly. (code -27)
    ! Error: Browsing on the local area network for iPhone von Mobile (3). Ensure the device is unlocked and attached with a cable or associated
      with the same local area network as this Mac.
      The device must be opted into Developer Mode to connect wirelessly. (code -27)

[✓] Network resources [720ms]
    • All expected network resources are available.

! Doctor found issues in 1 category.
```

</details>

**Comment by darshankawar** [comment]
Thanks for the detailed report @NilouMinooei 
It seems it is related to https://github.com/flutter/flutter/issues/164483 in which, in latest master version, it does show the focus on each item but after an extra silent focus before the item starts. In stable, the behavior is as reported by you in this issue. 

stable : 3.29.0

**Comment by chunhtai** [comment]
closed as a duplicate of https://github.com/flutter/flutter/issues/164483
